.pre-commit-config.yaml

```
@@ -94,3 +94,8 @@ repos:
       - id: taplo-format
         # See options: https://taplo.tamasfe.dev/configuration/formatter-options.html
         args: [--option, "reorder_arrays=true", --option, "reorder_keys=true"]
+
+  - repo: https://github.com/woodruffw/zizmor-pre-commit
+    rev: v1.0.0
```

```suggestion
    rev: v0.8
```

In the interest of moving this PR forward (and improving this repo's security) I think it's fine to revert to an older version. The cache-poisoning checks from the newer version can be addressed in a separate PR.